src/components/main-form.js

```diff
@@ -80,22 +81,44 @@ const AmortizationTable = styled(Table)`
     border-left-color: ${GRAY}};
   }
 `
+const emphasize = cx('card', css(`border-color: ${LIGHT_GREEN};`))
```

Comment:
renamed from `highlight` because it is not actually highlighting. Not sure if there's a better word